packages/contracts/contracts/voting/admin/Admin.sol

```diff
@@ -2,17 +2,15 @@
 
 pragma solidity 0.8.10;
 
-import {Counters} from "@openzeppelin/contracts/utils/Counters.sol";
+import {CountersUpgradeable} from "@openzeppelin/contracts-upgradeable/utils/CountersUpgradeable.sol";
 
-import {PluginCloneable} from "../../core/plugin/PluginCloneable.sol";
 import {IDAO} from "../../core/IDAO.sol";
+import {GovernancePluginCloneable} from "../../core/plugin/GovernancePluginCloneable.sol";
 
 /// @title Admin
 /// @author Aragon Association - 2022.
-/// @notice The admin address governance plugin giving execution permission on the DAO to a single address
-contract Admin is PluginCloneable {
-    using Counters for Counters.Counter;
-
+/// @notice The admin address governance plugin giving execution permission on the DAO to a single address.
+contract Admin is GovernancePluginCloneable {
```

Comment:
Here is where my concerns with the `Governance` name find a good example. I still find that this interface/base-contract applies to proposals. Governance is much closer to voting than to proposing things

Reply:
APP-1447 created :+1: